### PR TITLE
ACTUAL Release 3.4

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by maxulysse** [comment]
Replace #1320

**Review by LaurenceKuhl — APPROVED** [review]

**Review by mirpedrol — APPROVED** [review]
LGTM :)

**Comment by maxulysse** [comment]
Just waiting on #1325

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +64,7 @@ Ráhpajávvre is the Lule Sámi spelling of Rapaselet.
 
 - [#1246](https://github.com/nf-core/sarek/pull/1246) - Back to dev
 - [#1259](https://github.com/nf-core/sarek/pull/1259) - nf-prov plugin
+- [#1288](https://github.com/nf-core/sarek/pull/1288) - Add nf-test continuous integration.
```
Comment (maxulysse) [inline CHANGELOG.md:67]: Shouldn't this be in this release?

File: conf/modules/annotate.config [hunk 2]
```diff
@@ -37,7 +37,7 @@ process {
             ext.args         = { [
                 (params.vep_dbnsfp && params.dbnsfp && !params.dbnsfp_consequence)    ? "--plugin dbNSFP,${params.dbnsfp.split("/")[-1]},${params.dbnsfp_fields}"                                              : '',
                 (params.vep_dbnsfp && params.dbnsfp && params.dbnsfp_consequence)     ? "--plugin dbNSFP,'consequence=${params.dbnsfp_consequence}',${params.dbnsfp.split("/")[-1]},${params.dbnsfp_fields}"   : '',
-                (params.vep_loftee)                                                   ? "--plugin LoF,loftee_path:/opt/conda/envs/nf-core-vep-${params.vep_version}/share/ensembl-vep-${params.vep_version}-0" : '',
+                (params.vep_loftee)                                                   ? "--plugin LoF,loftee_path://usr/local/share/ensembl-vep-${params.vep_version}" : '',
```
Comment (maxulysse) [inline conf/modules/annotate.config:40]:
```suggestion
                (params.vep_loftee)                                                   ? "--plugin LoF,loftee_path:/usr/local/share/ensembl-vep-${params.vep_version}" : '',
```

File: subworkflows/local/samplesheet_to_channel/main.nf.test [hunk 3]
```diff
@@ -0,0 +1,34 @@
+nextflow_workflow {
+
+    name "Test Workflow SAMPLESHEET_TO_CHANNEL"
+    script "subworkflows/local/samplesheet_to_channel/main.nf"
+    workflow "SAMPLESHEET_TO_CHANNEL"
+
```
Comment (maxulysse) [inline subworkflows/local/samplesheet_to_channel/main.nf.test:6]: we're missing tags

File: modules.json [hunk 4]
```diff
@@ -7,469 +7,500 @@
...
-                        "git_sha": "911696ea0b62df80e900ef244d7867d177971f73",
+                        "git_sha": "8fc1d24c710ebe1d5de0f2447ec9439fd3d9d66a",
                         "installed_by": ["modules"]
                     },
                     "bcftools/mpileup": {
                         "branch": "master",
-                        "git_sha": "911696ea0b62df80e900ef244d7867d177971f73",
-                        "installed_by": ["modules"]
+                        "git_sha": "8fc1d24c710ebe1d5de0f2447ec9439fd3d9d66a",
+                        "installed_by": ["bam_ngscheckmate"]
```
Comment (maxulysse) [inline modules.json:27]: I'd say this one should come from modules as well, but yeah not a problem for the release.
I'm wondering what happened there, maybe an issue when I updated the subworkflow

File: CHANGELOG.md (hunk 1 above)
Comment (FriederikeHanssen) [reply CHANGELOG.md:70]: What do you mean? Actual integration is not implemented someone added I think the bare bones. From my side it should definitely it be in the release notes, but the autogenerated ones need to be carefully cleaned up in any case to make them readable

File: subworkflows/local/samplesheet_to_channel/main.nf.test (hunk 3 above)
Comment (maxulysse) [reply subworkflows/local/samplesheet_to_channel/main.nf.test:6]: but not needed in the release

File: CHANGELOG.md (hunk 1 above)
Comment (maxulysse) [reply CHANGELOG.md:70]: yeah, but it's listed in the 3.3.2 release

Comment (maxulysse) [reply CHANGELOG.md:70]:
```suggestion
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -5,6 +5,57 @@ All notable changes to this project will be documented in this file.
...
+Pårtetjåkko is a mountain in the south of the park.
+
+### Added
+
+- [#1113](https://github.com/nf-core/sarek/pull/1113) - Adding CNVkit genemetrics module
+- [#1193](https://github.com/nf-core/sarek/pull/1193) - Adding support for Sentieon's DnaScope for germline variant-calling including joint-germline
+- [#1244](https://github.com/nf-core/sarek/pull/1244) - Add bcf annotate module
+- [#1252](https://github.com/nf-core/sarek/pull/1252) - Added NGSCheckMate tool for checking that samples come from the same individual
+- [#1271](https://github.com/nf-core/sarek/pull/1271) - Back to dev
+- [#1290](https://github.com/nf-core/sarek/pull/1290) - Add nf-test for whole pipeline.
```
Comment (maxulysse) [inline CHANGELOG.md:19]:
```suggestion
- [#1288](https://github.com/nf-core/sarek/pull/1288) - Add nf-test continuous integration (but no tests)
- [#1290](https://github.com/nf-core/sarek/pull/1290) - Add nf-test for whole pipeline
```

**Review by maxulysse — APPROVED** [review]

**Comment by maxulysse** [comment]
Finally

**Pull request merged**